operations/service_web.go

```diff
@@ -48,6 +49,9 @@ func startWebService() cli.Command {
 		Action: func(c *cli.Context) error {
 			ctx, cancel := context.WithCancel(context.Background())
 
+			_, err := maxprocs.Set()
```

Comment:
Can we add a comment on what max procs is needed for? Otherwise it looks like a random/unnecessary bit of logic and future readers might wonder what's so important about it that it warrants `grip.EmergencyFatal`.